boltdesignsystem/bolt
# refactor(icon): change schema format Yaml => JSON

## Summary

This simply changes the schema format for icons from Yaml to JSON; not changes too content. 

## Details

We need to get access to the list of available icons for our work in #1207 before WebPack starts up and so converting this to JSON makes it available. I've also gone around to the other schema files that reference it and the build tasks that automatically updates it. I've chatted with @sghoweri and he says that Bolt is going to move away from Yaml Schema files soon anyway.

## How to test

- Change schema file
- On Icon Schema docs page note that it reflects changes
- Visit button schema docs page to see Icon schema updated
- Delete an icon name from schema file and restart build tools and note that the icon name is inserted back in
